src/server/debugcmd.h

```diff
@@ -19,13 +22,35 @@ class DebugCmd {
 
  private:
   void Populate(CmdArgList args);
-  void PopulateRangeFiber(uint64_t from, uint64_t len, std::string_view prefix, unsigned value_len);
+  void MemoryFuzz(CmdArgList args);
   void Reload(CmdArgList args);
   void Replica(CmdArgList args);
   void Load(std::string_view filename);
   void Inspect(std::string_view key);
   void Watched();
 
+ public:
```

Comment:
Is there a reason why you added another section of public and private in this class and not add to the existing ?

Reply:
That's still just experimental code, not ready for any polishment 😅